.vscode/settings.json

```diff
@@ -1,6 +1,7 @@
 {
   "python.formatting.provider": "black",
-  "python.linting.mypyEnabled": true,
+  "python.linting.mypyEnabled": false,
```

Comment:
Yes, I believe pyright is much better than mypy because pyright makes some inferences, mypy does not make any inference and is pretty annoying sometimes

Reply:
I left it in the precommit nonetheless